### DISCUSSION TITLE
does it support for using vue-flow component as a node to make nested vue-flow

### DISCUSSION THREAD

**Discussion by leonaaaardo** [bcakmakoglu/vue-flow#1340] [Q&A]
as above, does the vue-flow lib support for using vue-flow component as a node, then drag it into the dropzone to make nested vue-flow chart?

**Comment by bcakmakoglu** [comment]
I recommend *not* attempting this.
It's going to mis-align your edges as soon as the nested Flow has a zoom value that is not `1`, given that one flow component is *inside* another, i.e. **this will not work correctly:**

```vue
<VueFlow>
	<template #node-flow="props">
		<div>
			<VueFlow>
				...
```

What you *can* do is move the "nested" flow *outside* the tree of the other flow, i.e. *this does work*

```vue
<VueFlow>
	<template #node-flow="props">
		<Teleport to="body">
			<div>
				<VueFlow>
					...
